libr/anal/op.c

```diff
@@ -86,12 +86,70 @@ static int defaultCycles(RAnalOp *op) {
 	}
 }
 
+static int mask_anal_to_arch (int m) {
+	int r = R_ARCH_OPTION_SIZE; // R_ANAL_OP_MASK_BASIC
+	r |= R_ARCH_OPTION_ANAL;
+	if (m & R_ANAL_OP_MASK_ESIL) {
+		r |= R_ARCH_OPTION_ESIL;
+	}
+	if (m & R_ANAL_OP_MASK_DISASM) {
+		r |= R_ARCH_OPTION_CODE;
+	}
+	return r;
+#if 0
```

Comment:
remove

Reply:
this is totally WIP because the usage of this api is wrong in many places